src/megaapi_impl.cpp

```diff
@@ -19724,6 +19718,13 @@ void MegaApiImpl::sendPendingRequests()
             bool resumeProcess = (request->getParamType() == 1);   // resume existing ephemeral account
             bool cancelProcess = (request->getParamType() == 2);
 
+            if (request->getParamType() < mega::MegaApi::AFFILIATE_TYPE_INVALID
```

Comment:
We want to provide backwards compatibility. Apps can still create accounts without providing the new parameters, as you still allow the old API without the new params. In consequence, if no public handle is provided, the request should not fail. I suggest to add the `!ISUNDEF(lastPublicHandle)`, something like:
```
if (!ISUNDEF(lastPublicHandle) 
    && lastPublicHandleType <= mega::MegaApi::AFFILIATE_TYPE_INVALID
    && lastPublicHandleType > mega::MegaApi::AFFILIATE_TYPE_CONTACT)
```